Add guide to install istio on Huawei CCE (#9051)
istio/istio.io: content/en/docs/setup/platform-setup/huaweicloud/index.md

---
title: Huawei Cloud

> description: Instructions to setup an Huawei Cloud kubernetes cluster for Istio.

Reviewer: ```suggestion
description: Instructions to setup an Huawei Cloud Kubernetes cluster for Istio.
```

weight: 23
skip_seealso: true
aliases:
    - /docs/setup/kubernetes/prepare/platform-setup/huaweicloud/
    - /docs/setup/kubernetes/platform-setup/huaweicloud/
keywords: [platform-setup,huawei,huaweicloud,cce]
owner: istio/wg-environments-maintainers
test: no
---

Follow these instructions to prepare a cluster for Istio using the
[Huawei Cloud Container Engine
](https://www.huaweicloud.com/intl/product/cce.html).
You can deploy a Kubernetes cluster to Huawei Cloud quickly and easily in the
`Cloud Container Engine Console`, which fully supports Istio.

{{< tip >}}
Huawei offers a {{< gloss >}}managed control plane{{< /gloss >}} add-on for the Huawei Cloud Container Engine,
which you can use instead of installing Istio manually.
Refer to [Huawei Application Service Mesh](https://support.huaweicloud.com/istio/)
for details and instructions.
{{< /tip >}}

> Following the [Huawei Cloud Instructions](https://support.huaweicloud.com/en-us/qs-cce/cce_qs_0008.html) to prepare a cluster before manually installing Istio, proceed as follows:

Reviewer: Should the user follow the instructions in the link? I'm guessing not, but it's a little confusing.

Author: This is an instruction about how to create a k8s using huawei cce

1.  Log in to the CCE console. Choose **Dashboard** > **Buy Cluster** to open the **Buy Hybrid Cluster** page. An alternative way to open that page is to choose **Resource Management** > **Clusters** in the navigation pane and click **Buy** next to **Hybrid Cluster**.

1.  On the **Configure Cluster** page, configure cluster parameters.
    In this example, a majority of parameters retain default values. After the cluster configuration is complete, click Next: **Create Node** to go to the node creation page.

    {{< tip >}}

>     Istio release has some requirement for the Kubernetes version,
>     select the version according to Istio's [support policy]](/about/supported-releases#supported-releases/).

Reviewer: ```suggestion
    Every Istio release has a [supported list](/about/supported-releases#supported-releases/) of Kubernetes versions. Select the Kubernetes version that matches your desired Istio version.
```

          istio.io/rev: default
          operator.istio.io/component: IngressGateways
          operator.istio.io/managed: Reconcile
          operator.istio.io/version: 1.9.0
          release: istio
        name: istio-ingressgateway
        namespace: istio-system
      spec:
        clusterIP: 10.247.7.192
        externalTrafficPolicy: Cluster
        loadBalancerIP: 119.8.36.132     ## ELB EIP
        ports:
        - name: status-port
          nodePort: 32484
          port: 15021
          protocol: TCP
          targetPort: 15021
        - name: http2
          nodePort: 30294
          port: 80
          protocol: TCP
          targetPort: 8080
        - name: https
          nodePort: 31301
          port: 443
          protocol: TCP
          targetPort: 8443
        - name: tcp
          nodePort: 30229
          port: 31400
          protocol: TCP
          targetPort: 31400
        - name: tls
          nodePort: 32028
          port: 15443
          protocol: TCP
          targetPort: 15443
        selector:
          app: istio-ingressgateway
          istio: ingressgateway
        sessionAffinity: None
        type: LoadBalancer
      EOF
      {{< /text >}}

> 1.  Play with Istio and try the examples provided officially.

Reviewer: ```suggestion
Start playing with Istio by trying out the various [tasks](/docs/tasks).
```

---
title: Huawei Cloud
description: Instructions to setup an Huawei Cloud kubernetes cluster for Istio.

> weight: 25

Reviewer 2: The weight should be between 20 and 25 as those are GKE and IBM. Maybe 23?

        istio.io/rev: default
        operator.istio.io/component: IngressGateways
        operator.istio.io/managed: Reconcile
        operator.istio.io/version: 1.9.0
        release: istio
      name: istio-ingressgateway
      namespace: istio-system
    spec:
      clusterIP: 10.247.7.192
      externalTrafficPolicy: Cluster
      loadBalancerIP: 119.8.36.132     ## ELB EIP
      ports:
      - name: status-port
        nodePort: 32484
        port: 15021
        protocol: TCP
        targetPort: 15021
      - name: http2
        nodePort: 30294
        port: 80
        protocol: TCP
        targetPort: 8080
      - name: https
        nodePort: 31301
        port: 443
        protocol: TCP
        targetPort: 8443
      - name: tcp
        nodePort: 30229
        port: 31400
        protocol: TCP
        targetPort: 31400
      - name: tls
        nodePort: 32028
        port: 15443
        protocol: TCP
        targetPort: 15443
      selector:
        app: istio-ingressgateway
        istio: ingressgateway
      sessionAffinity: None
      type: LoadBalancer
    EOF
    {{< /text >}}

1.  Play with Istio and try the examples provided officially.

> 

Reviewer 2: Extra empty lines.

Author: thanks
